src/Filters/FiltersGroup.php

```
@@ -0,0 +1,59 @@
+        foreach ($members as $member) {
+            if (! $member instanceof AllowedFilter) {
+                throw new InvalidArgumentException(
+                    'FiltersGroup members must be AllowedFilter instances, got '.get_debug_type($member).'.'
+                );
+            }
+        }
+
+        $this->members = $members;
+    }
+
+    public function __invoke(Builder $query, mixed $value, string $property): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$property'. (undefined)

(UnusedFormalParameter)